includes/Dashboard.php

```diff
@@ -180,10 +180,17 @@ public function enqueue_assets( $hook_suffix ) {
 			WEBSTORIES_VERSION
 		);
 
+		wp_register_style(
```

Comment:
Why load this as another CSS file. You are loading google sans a couple of lines above. 
You can load them all at once. 
```
wp_register_style(
			'google-fonts',
			'https://fonts.googleapis.com/css?family=Google+Sans|Google+Sans:b|Google+Sans:500|Roboto:ital,wght@0,400&display=swap',
			[],
			WEBSTORIES_VERSION
		);
```

Reply:
thanks for this! To answer the why: because i was following the other patterns I saw for registering styles and because I do not normally write PHP. I'll update.

Reply:
This is less of a php question and more of a css and performance question. I was wonder if there was a good reason to load them as two files. One as the swap param on it. 

There is overhead of two http requests, making it one will help performance and seeing the font is pretty important.